src/protocols/bitcoin/views/TransactionDetails.vue

```diff
@@ -72,21 +72,28 @@ export default defineComponent({
 
     const transactionOwner = route.params.transactionOwner as string;
 
+    const bitcoinAdapter = ProtocolAdapterFactory.getAdapter(PROTOCOL_BITCOIN);
+
     const transaction = ref<ITransaction>();
 
     const explorerUrl = computed(
-      () => ProtocolAdapterFactory
-        .getAdapter(PROTOCOL_BITCOIN)
-        .getExplorer()
-        .prepareUrlForHash(transaction.value?.hash),
+      () => bitcoinAdapter.getExplorer().prepareUrlForHash(transaction.value?.hash),
     );
 
     const transactionFee = computed((): number => transaction.value?.tx?.fee ?? 0);
 
     const totalAmount = computed((): number => transaction.value?.tx
-      ? getTxAmountTotal(transaction.value, transaction.value.tx.senderId !== transactionOwner)
+      ? bitcoinAdapter
+        .getTxAmountTotal(transaction.value, transaction.value.tx.senderId !== transactionOwner)
       : 0);
 
+    const amount = computed(
```

Comment:
This is a thing that I already changed within another PR. For protocols different than the AE the amount should not be calculated again and again. The `transaction.tx.amount` is the real amount.